README.md

```diff
@@ -1,33 +1,22 @@
-
-
-## All PRs should be opened against master branch. This branch is being retired and will be deleted.
-
-### Microsoft Azure SDK for .NET
+# Microsoft Azure SDK for .NET
  ----------
-The Microsoft Azure SDK for .NET allows you to build applications
-that take advantage of scalable cloud computing resources.
+Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It’s the freedom to build, manage, and deploy applications on a massive, global network using your favorite tools and frameworks.
+
+This repo contains the libraries to allow you to easily leverage Azure from your applications and tools.
 
-### Download Packages
+## Package List
 
-For a full list of packages available for download in this repository, please see our [list of .NET SDK packages](https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/Documentation/sdk-for-net-packages.md).
+For a full list of packages built from this repository, please see our [list of .NET SDK packages](https://github.com/Azure/azure-sdk-for-net/tree/master/Documentation/sdk-for-net-packages.md).
 
 ### Target Frameworks:
 
-* .NET Framework 4.5.2
-* Netstandard 1.4, based on the NetCore framework
+* .NET Framework 4.6.1
```

Comment:
We need to be careful about changing this all up for the entire repo. @shahabhijeet when do you plan to move to these targets for management?

Reply:
@shahabhijeet how do you think we should message this to be clear that new libraries and major (possibly minor) rev's of the library should change their baseline and that this is the baseline moving forward?

Reply:
Note, I can remove these changes to get the rest of it through if thats better. We do need to figure this one out too though.

Reply:
Yeah I'd suggest being careful about this part of your change as we need to understand the plan and then message it.